### PR TITLE
update readme with hsf cookiecutter readme template

### PR DESCRIPTION
**Pull request by amanmdesai**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to respond to your contribution.
To ensure that the right people get notified, you can tag some of the last contributors with `@githubname`.

Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact the HSF training convenors ([contacts here](https://hepsoftwarefoundation.org/workinggroups/training.html)).

You may delete these instructions from your pull request.

\- HSF Training
</details>

### CODE REVIEW

**Comment by wdconinc** [comment]
No rebase button... @amanmdesai can you rebase or merge gh-pages into this branch?

**Comment by amanmdesai** [comment]
> No rebase button... @amanmdesai can you rebase or merge gh-pages into this branch?

Done. Thanks.

**Comment by wdconinc** [comment]
I could review but was going to let @klieret comment.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,95 @@
...
+[![Upcoming Events][schools-badge]][schools]
+[![Twitter Follow][twitter-badge]][twitter]
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/hsf-training/hsf-training-cicd-basics-webpage/main.svg)](https://results.pre-commit.ci/latest/github/hsf-training/hsf-training-cicd-basics-webpage/main)
+[![pages-build-deployment](https://github.com/hsf-training/hsf-training-cicd-basics-webpage/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/hsf-training/hsf-training-cicd-basics-webpage/actions/workflows/pages/pages-build-deployment)
+
+# Title of the lesson
```
Comment (klieret) [inline README.md]: <-

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by klieret** [comment]
The thing is that this repository is currently not really used. It was meant to contain the intersection between the github and gitlab trainings, but we never really figured out a good way to combine both of these trainings (so this is now certainly already out of sync)...

File: README.md (hunk 1 above)
Comment (amanmdesai) [reply README.md]: Thanks. Implemented.

**Comment by amanmdesai** [comment]
> The thing is that this repository is currently not really used. It was meant to contain the intersection between the github and gitlab trainings, but we never really figured out a good way to combine both of these trainings (so this is now certainly already out of sync)...

I see, thanks. I did not notice that it was out of sync for that reason.

**Pull request closed without merging**